Assets/MixedRealityToolkit/SpatialMapping/Scripts/TapToPlace.cs

```diff
@@ -275,89 +274,5 @@ private static Vector3 GetGazePlacementPosition(Vector3 headPosition, Vector3 ga
             }
             return headPosition + gazeDirection * defaultGazeDistance;
         }
-
-        #region IInputSource Implementation
```

Comment:
@StephenHodgson I just noticed I had forgot to remove this unused code so I pushed one last commit.